### ISSUE TITLE
replace codes with relations (strings with things)

### ISSUE THREAD

**Issue by VladimirAlexiev** [Interoperable-data/ERA-Ontology-3.1.0#118]
[A tunnel](https://prod.virtuoso.ecdp.tech.ec.europa.eu/describe/?url=http://data.europa.eu/949/functionalInfrastructure/tunnels/4cf1a1f13306621a1066abb47655219f1e3e624d) has a data prop `imCode` Infrastructure manager (IM)'s code.
But that is wrong modeling: the tunnel should have a **relation** to its infrastructure manager, which on its part should have the ID code.

In contrast, the "in country" prop is better because it's a relation to the country resource.

The same problem probably pertains to:
- all kinds of infrastructure
- other codes that should be converted to relations

**Comment by gatemezing** [comment]
Yes, the new version will tackle an organization pattern for everything related to ORG, role and this will solve this issue. More to come
